lib/cmd/common/import.js

```diff
@@ -32,8 +32,9 @@ function imports (argv, cb) {
     var appTitle = args[1];
     var appTemplateType = args[2];
     var fileOrRepo = args[3];  // note this is optional, can be null
+    var repoBranch = args[4] || 'master';  // note this is optional, will default to master
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComment)